main.go

```diff
@@ -50,10 +50,11 @@ type Proxy struct {
 	ManagementAPI string    `required:"true" split_words:"true"`
 
 	// optional params
-	CookieName    string `default:"_auth_proxy" split_words:"true"`
-	ReturnToParam string `default:"returnTo" split_words:"true"`
-	TokenParam    string `default:"token" split_words:"true"`
-	TokenPath     string `default:"/auth/token" split_words:"true"`
+	CookieName       string `default:"_auth_proxy" split_words:"true"`
+	ReturnToParam    string `default:"returnTo" split_words:"true"`
+	RobotsTxtDisable bool   `default:"false" split_words:"true"`
```

Comment:
What does the `split_words` here do?

Reply:
These struct tags (`default` and `split_words`) are read by a [library](https://github.com/kelseyhightower/envconfig) we use for reading environment variables. `split_words:"true"` is required for variable names that use an underscore to separate words.